libs/platform/src/lib/table/components/table-row/table-row.component.html

```diff
@@ -1,228 +1,258 @@
-<td
-    *ngIf="_fdpTableService._semanticHighlighting$ | async as semanticHighlighting"
-    fd-table-cell
-    fd-table-status-indicator
-    fdkDisabled
-    [addDisabledClass]="false"
-    [status]="row.value[semanticHighlighting]"
-    [cellFocusedEventAnnouncer]="_itemFocusedEventAnnouncer"
-></td>
-
-<!-- hidden text for screenreader -->
-<ng-template #selectionScreenReaderText>
-    <span
-        aria-hidden="true"
-        [attr.id]="_rowSelectionHelperTextId"
-        [ngStyle]="{
-            position: 'absolute !important',
-            height: '1px',
-            width: '1px',
-            overflow: 'hidden',
-            clip: 'rect(1px, 1px, 1px, 1px)',
-            color: 'transparent',
-            display: 'block'
-        }"
-    >
-        {{ (row.checked ? 'platformTable.deselectSingleRow' : 'platformTable.selectSingleRow') | fdTranslate }}
-    </span>
+<ng-template #fallbackContent>
+    <fd-busy-indicator [loading]="true" size="s" [block]="true"> </fd-busy-indicator>
```

Comment:
I think it would be best to render nothing here, seeing the busy indicator to the left of the table for a fraction of a second is a bit weird. I think rendering nothing will make the enhanced rendering less perceptible

Reply:
Agree, but our UX wants to see a placeholder. You can see it fast there in ngx playground, but application may be different (ex: api, other computations..., which ofc should be addressed by our side) Initially, I want to use `fd-skeleton`, but some implementation in that component make the CPU spikes up...
I guess I can let it decided by application to pass the placeholder.